modules/structs/repo.go

```diff
@@ -93,6 +93,7 @@ type Repository struct {
 	AvatarURL                 string           `json:"avatar_url"`
 	Internal                  bool             `json:"internal"`
 	MirrorInterval            string           `json:"mirror_interval"`
+	RepoTransfer              *RepoTransfer    `json:"repo_transfer"`
```

Comment:
Why we need this?

Reply:
To have a possibility for clients to check if the repo is currently waiting for an approval to transfer it